File: Behavioral/ChainOfResponsibility/Java/Manager.java
// Manager.java

public class Manager extends SupportHandler {
  @Override
  public void handleRequest(String request) {
    if (request.equals("Advanced Issue")) {
      System.out.println("Manager: Handling request - " + request);
    } else {
      System.out.println("No handler available for - " + request);
    }
  }
}

File: Behavioral/ChainOfResponsibility/JavaScript/Manager.js
// Manager.js

const SupportHandler = require("./SupportHandler");

class Manager extends SupportHandler {
  handleRequest(request) {
    if (request === "Advanced Issue") {
      console.log(`Manager: Handling request - ${request}`);
    } else {
      console.log(`No handler available for - ${request}`);
    }
  }
}

module.exports = Manager;
